src/__tests__/DeepLink.test.js

```diff
@@ -26,42 +26,79 @@ describe('parseDeepLink', function () {
 
   describe('requestAddress', () => {
     makeLinkTests({
-      'edge://reqaddr?codes=eth-btc&post=testpost.com': {
+      'edge://reqaddr?codes=eth-btc&post=https://bitwage.com&redir=https://bitwage.com/getaddr&payer=bitwage': {
```

Comment:
The `post=` and `redir=` params should be uuencoded before being presented as a Request for Address URI. Otherwise they may not parse properly. I realized that I don't have a clear example of this in the docs ☹️

Reply:
The app would then have to uudecode the `post=` and `redir=` params before using them.